app/(pages)/content/[collection]/edit/_components/CreateNew/CreateNew.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
You don't need to import React inside of Next.js ever. I think they do this for you implicitly somewhere